src/main/java/teammates/common/datatransfer/questions/FeedbackQuestionDetails.java

```diff
@@ -111,6 +111,14 @@ public String getQuestionText() {
     }
 
     /**
+     * Returns a list of strings where each string is an instruction to answer the question.
+     * This will be displayed in the submission page just above the responses.
```

Comment:
This should not be there. The method has no business as to where the instructions will be displayed.